spec/models/comment_spec.rb

```diff
@@ -765,4 +829,36 @@
       end
     end
   end
+
+  describe "#mark_as_spam!" do
+    let(:comment) { create(:comment, approved: 1, spam: 0) }
+
+    before do
+      # Guard clause; emails shouldn't send in test env, but including just to be safe
```

Comment:
emails...? this would at most go to akismet and there it's fine to rely on the environment check in the model. Even if that was removed the config doesn't have a valid AKISMET_KEY so nothing can actually happen. So this guard clause is redundant